### PR TITLE
Remove sshd installer from default stacks

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Remove sshd installer from default stacks

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7685

#### Release Notes
Remove sshd installer from default stacks

#### Docs PR
n/a

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

**Review by garagatyi — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-test

**Review by riuvshin — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by benoitf** [comment]
I'm not sure it's solving the expected issue.
If user enable the ssh agent it will then expect that he will have ssh access. So removing it by default is not solving the issue.

If I select "ssh agent", I want to be able to connect through ssh to my workspace. It doesn't mean that I'm requiring to have sshd installed.

On OpenShift.io ssh access is possible through ssh keys
https://developers.openshift.com/managing-your-applications/remote-connection.html

**Review by ashumilova — APPROVED** [review]

File: selenium/che-selenium-test/src/test/resources/templates/workspace/docker/eclipse_php.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
           },
           "installers": [
             "org.eclipse.che.terminal",
-            "org.eclipse.che.ws-agent",
-            "org.eclipse.che.ssh",
+            "org.eclipse.che.ws-agent"
```
Comment (musienko-maxim) [inline selenium/che-selenium-test/src/test/resources/templates/workspace/docker/eclipse_php.json]: after 'ws-agent' was missed coma

**Comment by skabashnyuk** [comment]
@benoitf AFAIK the feature that you pointed me is for OpenShift2, OpenShift 3 has no such feature. 
According to this https://docs.openshift.com/container-platform/3.3/creating_images/guidelines.html using sshd is not recommended. We investigating possible alternatives.

**Comment by skabashnyuk** [comment]
ci-test

**Review by artaleks9 — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-build

**Pull request merged**